src/pages/docs/guides/chat/bring-own-database.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: "Guide: Store chat data in your own systems"
+meta_description: "Learn how to store chat data in your own systems with Ably."
+meta_keywords: "chat, data, storage, Ably, chat SDK, realtime messaging, dependability, cost optimisation"
+---
+
+Ably Chat is designed to be a simple and easy to use realtime chat solution that handles any scale from 1:1 and small group chats to large livestream chats with millions of users.
+
+It is designed to be a reliable building block for your application. One trade-off we make is the data retention duration for chat messages. It is 30 days by default, but can be increased up to a year on request.
+
+This guide presents different ways to use Ably Chat and store chat data in your own systems, which can help you meet your data retention requirements, as well as help you build more complex use cases such as search, analytics, and more.
+
+## Different ways to export data from Ably Chat
+
+We will explain each in detail, and provide code examples for each. This is an overview of the different ways to export data from Ably Chat.
+
+This article covers the following options:
```

Comment:
I think there's a fifth option here - there's nothing to say someone couldn't paginate the history API in some circumstances.